extensions/neo4j/deployment/src/main/java/io/quarkus/neo4j/deployment/DevServicesBuildTimeConfig.java

```diff
@@ -29,4 +30,18 @@
      */
     @ConfigItem
     public Map<String, String> additionalEnv;
+
+    /**
+     * This value will be used to expose the bolt-port of the container to the outside. It must be a free port,
+     * otherwise startup will fail. The dev services will try to allocate a free port by default.
+     */
+    @ConfigItem
+    public OptionalInt fixedBoltPort = OptionalInt.empty();
```

Comment:
I think the `fixed` is implicit in the config? How about simply `boltPort` and `httpPort`?

Reply:
Aaa yes, why not? I just followed this https://github.com/quarkusio/quarkus/blob/main/extensions/devservices/postgresql/src/main/java/io/quarkus/devservices/postgresql/deployment/PostgresqlDevServicesProcessor.java#L84 but didn't notice the config actually don't use the notion of "fixed"

Reply:
Done.